docs/participant_information.md

```diff
@@ -4,70 +4,67 @@ nav_title: Logistics
 ---
 
 
-**Coming soon:** Find helpful information related to health and safety and the workshop location.
-
+Find helpful information about the workshop location, travel reimbursement, and healthy and safety.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-<!--
-
 ### Table of Contents
 
-- [COVID-19 Safety Policy](#covid-19-safety-policy)
+- [Action Items](#action-items)
 - [Workshop Location](#workshop-location)
-  - [Parking](#parking)
-  - [Nearby Lodging](#nearby-lodging)
 - [Travel Reimbursement](#travel-reimbursement)
-
--->
+- [Health and Safety](#health-and-safety)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-<!--
+## Action Items
```

Comment:
Maybe, "Tasks to complete before the workshop"? I'm not sure "action items" conveys this as clearly?